components/selection/README.md

```diff
@@ -152,8 +152,8 @@ The `d2l-selection-action` is an optional component that provides a button for a
 
 The `d2l-selection-action-dropdown` is an optional component that provides a button opener for dropdown content associated with the selection component (ex. bulk actions). The `requires-selection` attribute may be specified to indicate that the opener should be non-interactive if nothing is selected.
 
-<!-- docs: demo live name:d2l-selection-action-dropdown -->
-```html
+<!-- docs: demo live name:d2l-selection-action-dropdown autoSize:false size:medium -->
```

Comment:
You likely want to add `align:baseline` to both of these so it places the opener at the top rather than middle